bindgen-cli/main.rs

```diff
@@ -40,8 +40,13 @@ pub fn main() {
                 eprintln!("{}", info);
             }));
 
-            let bindings =
-                builder.generate().expect("Unable to generate bindings");
+            let bindings = match builder.generate() {
+                Ok(bindings) => bindings,
+                Err(err) => {
+                    eprintln!("Unable to generate bindings: {err}");
```

Comment:
This feels a little drive-by?

Reply:
I can move those to different PRs if you want